datascroller/scroller.py

```diff
@@ -53,9 +53,18 @@ def get_window_string(self):
     def get_location_string(self):
         """Creates a header showing where the top left df corner is"""
         # TODO (baogorek): Handle thousands and millions
```

Comment:
Formatting update according to autopep-8 standard. Let's decide if this is too strict.

Reply:
I don't have a problem changing existing code to pep-8. Whether we demand new code adhere to it can be discussed.